src/nsls2api/infrastructure/config.py

```diff
@@ -76,4 +82,9 @@ def get_settings() -> Settings:
 
     :returns: The dictionary of current settings.
     """
-    return Settings()
\ No newline at end of file
+    if os.environ.get("PYTEST_VERSION") is not None:
+        settings = Settings(_env_file=str(Path(__file__).parent.parent / "pytest.env"))
```

Comment:
This is gentler on my eyes...

```suggestion
        PROJ_SRC_PATH = Path(__file__).parent.parent
        test_env_file = str(PROJ_SRC_PATH / "pytest.env")
        settings = Settings(_env_file=test_env_file)
```

Reply:
If this approach is agreeable, consider making `PROJ_SRC_PATH` a (non-exported) global in this module and reusing it above in 

```
    model_config = SettingsConfigDict(
        env_file=...
    )
```